### DISCUSSION TITLE
zh-cn/posts/2021/05/06/long-hair/

### DISCUSSION THREAD

**Discussion by giscus** [CyrusYip/blog-comments#80] [Comments]
# zh-cn/posts/2021/05/06/long-hair/

 缘起 # 高三那年我诊断出强迫症和焦虑症，症状是一学习就咳嗽1得很厉害。于是没去上学，平时就在学校附近的出租屋自己复习。有天刷知乎看到讲男生留头发的帖子，我看照片还挺帅气啊。干脆我也试试看吧，反正也不用去上课了，不用在意别人的目光。我留长发有三个原因：感觉好看、懒得打理头发、懒得去理发店。上网买了个帽子，就这样开始留头发了。
留头发有三个阶段：能打理、不能打理、成功。
能打理
头发还没特别长的时候我就留侧分，再长一些就留背头了。 不能打理
头发长到遮眼的时候没办法弄发型了，这时候就直接戴帽子盖住这乱糟糟的头发。平时在家就戴发箍，洗了头之后也戴。度过这个阶段就成功啦。 歧视 # 打理头发是留头发过程中最简单的事情了，可怕的是被歧视。例如：

https://cyrusyip.org/zh-cn/posts/2021/05/06/long-hair/

**Comment by Liam-Zhong** [comment]
长发好看哩
